app/modules/profile/templates/profile/edit.html

```diff
@@ -119,7 +119,23 @@ <h4 class="alert-heading"><i class="align-middle" data-feather="x-circle"></i> E
 
         </div>
 
+
+    </div>
+    <h1></h1>
```

Comment:
Quitar la etiqueta _h1_ vestigial y añadir un _hr_ en su lugar para hacer una separación más estética
```suggestion
    <hr>
```